package.json

```diff
@@ -95,11 +95,14 @@
     "pixi.js": "^5.x.x",
     "prop-types": "^15.x.x",
     "react": "^16.x.x",
-    "react-dom": "^16.x.x"
+    "react-dom": "^16.x.x",
+    "gsap": "^2.x.x"
   },
   "dependencies": {
     "@testing-library/react": "8.0.5",
+    "deepmerge": "^4.1.1",
     "fbjs": "^1.0.0",
+    "gsap": "^3.0.1",
```

Comment:
can this be moved to `peerDependencies`?

Reply:
Yeah, I had actually put it on peer dep to fix the build, but forgot to remove it from here :) Done now

Reply:
👍